src/main/java/ca/corefacility/bioinformatics/irida/service/analysis/workspace/galaxy/AnalysisCollectionServiceGalaxy.java

```diff
@@ -167,6 +186,13 @@ public CollectionResponse uploadSequenceFilesPaired(
 				description.addDatasetElement(pairedElement);
 			}
 		}
+		if(iridaTemporaryFileForward != null) {
+			iridaFileStorageUtility.cleanupDownloadedLocalTemporaryFiles(iridaTemporaryFileForward);
+		}
+
+		if(iridaTemporaryFileReverse != null) {
+			iridaFileStorageUtility.cleanupDownloadedLocalTemporaryFiles(iridaTemporaryFileReverse);
+		}
```

Comment:
Something feels wrong with the logic here.  We've got a for loop above making the `iridaTemporaryFile*`, but only 1 call cleaning them up.  Should that be dealt with differently?

Reply:
Looking at it now it does seem incorrect. Looks like I had a brain cramp. Will get it fixed up

Reply:
Updated to cleanup a list of irida temporary files created by the for loops for `uploadSequenceFilesSingleEnd` and `uploadSequenceFilesPaired` in 55799501f82e13585f8b7a670ee3feace53f349c

Reply:
Updated as discussed in ed329514516fc78562b1070af2fde122f529ae42